cookbooks/mantle_convection_with_continents_in_annulus/doc/mantle_convection_in_annulus.md

```diff
@@ -0,0 +1,89 @@
+depths. We use the geodynamic World Builder {cite:t}`Fraters2019c` to
+set the initial temperature and compositional field distribution
+(see {numref}`fig:init_visc`, {numref}`fig:init_compositions` and  {numref}`fig:init_temperature`).
+
+
+```{figure-md} fig:init_visc
+<img src="viscosity.*" width="90%" />
+
+the initial viscosity field.
+```
+
+```{figure-md} fig:init_compositions
+<img src="continents.*" width="90%" />
+
+The initial position of the three continents.
+```
+
+```{figure-md} fig:init_temperature
+<img src="temperature.*" width="90%" />
+
+The initial temperature field.
+```
+
+Similar to {cite:t}`ulvrova:etal:2019`, we use viscous rafts as `continents' to aid with
+modelling one-sided subduction systems. We use three continents of
+5000, 5000 and 3000 km length, covering roughly 30% of the model
+surface. To avoid subduction of the continents, we assign the viscous
+rafts a reference density of 2916 kg/m3, which corresponds to a 400
+kg/m3 density contrast with the reference mantle density.
+Furthermore, to avoid deformation of the continents, we assign
+them a viscosity that equals the maximum cut-off viscosity in the model,
+i.e. 6e24 Pa s. We also configure three subduction zones
+in the initial set-up to seed the model with initial negative
+buoyancy.
+
+As the goal of the publication is to investigate
+how average slab sinking rates relate to the vigour of mantle convection and mixing,
+various rheologies are considered in the lower mantle, as
+show in {numref}`fig:visc-profiles`.
+We will focus on the 'R' (reference) profile in this cookbook (green line).
+
+```{figure-md} fig:visc-profiles
+<img src="viscosity_profiles.*" width="90%" />
+
+Radial viscosity profiles for the three models discussed in the original
+publication (solid lines)
+compared to viscosity profiles as used in other studies.
+See original publication for the references corresponding to the
+dashed lines.
+```
+
+After 500 Myr we see multiple subductions taking place as well as
+many plumes/upwellings, as shown in {numref}`fig:evolution`
+
+```{figure-md} fig:evolution
+<img src="evolution.*" width="90%" />
+
+Results of the reference model (R) after 500 Ma of mantle convection simulation. Snapshots of the model at t = 500 Ma are shown for the viscosity (a) and
+temperature (b) within the modelled domain, showing five slabs actively subducting below the continents (pink). c Solid lines indicate average surface velocity (blue)
+and mobility ratio M (Tackley, 2000) (red) of model R as well as their dashed time-averages Vsurf=2.1 cm/a and M = 2.218. d Radial velocity of all tracers defined as
+slabs plotted at their position in the model where blue indicates sinking slabs and red slowly rising. Shown tracers are 300 K colder than the radial averaged
+temperature at similar depth and automatically obtained (see methods section of the publication).
+e 2D-histogram showing the tracer depth vs the radial velocity in 25 km by 2 mm/a bins. The
```

Comment:
The radial velocity in the plot looks like a rather narrow array of values than the min and max bounds of your x-axis. Would it be appropriate to scale those to see more of what's happening in that plot? Also I was curious what does this sub-plot show in a conceptual way (i.e. why do people look at plots like this).

Reply:
this plot is a screen capture of the paper. I cannot modify it...